### PR TITLE
Add ERC: ERC-1155 Multi-Asset extension

### PR DESCRIPTION
**Pull request by haruu8**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/eip-draft_title_abbrev.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
```
Comment (abcoathup) [inline ERCS/eip-draft_title_abbrev.md:1]:
```suggestion
---
eip: 7603
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

File: ERCS/eip-draft_title_abbrev.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+title: Context-Dependent Multi-Asset Tokens, EIP-1155 Extension
```
Comment (abcoathup) [inline ERCS/eip-draft_title_abbrev.md:2]:
```suggestion
title: Context-Dependent Multi-Asset Tokens, ERC-1155 extension
```

**Comment by haruu8** [comment]
@abcoathup 
Thank you for pointing out some mistakes; I modified the code based on your provided information. I'll be glad if you recheck the PR.

**Review by SamWilsn — APPROVED** [review]
Fix these at some point, but this looks good for a Draft.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**